storage/norcow_blockwise.h

```diff
@@ -236,9 +262,8 @@ secbool norcow_update_bytes(const uint16_t key, const uint8_t *data,
   }
 
   if (norcow_write_buffer_key == -1) {
-    memset(norcow_write_buffer, 0, sizeof(norcow_write_buffer));
+    memset(norcow_write_buffer, 0xFF, sizeof(norcow_write_buffer));
     norcow_write_buffer_key = key;
-    norcow_write_buffer[0] = NORCOW_VALID_FLAG;
```

Comment:
Here too, we should always write the `NORCOW_VALID_FLAG` and not rely on the identity with the memset. The compiler will optimize the code anyway if the two are identical.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/de177fd1aa86751e05d2fae01e647f4937054315